src/twisted/web/http.py

```diff
@@ -272,6 +294,9 @@ def parse_qs(qs, keep_blank_values=0, strict_parsing=0):
     """
     Like C{cgi.parse_qs}, but with support for parsing byte strings on Python 3.
 
+    This was created to help with Python 2 to Python 3 migration.
```

Comment:
I have added this comment to trigger a conversation.

Maybe this function should be deprecated (in a separate PR), and just use stdlib.

Reply:
Could we just replace it with a deprecated alias of `parse_qs`, or does its behavior differ significantly?

In any case yeah eventually deprecating it sounds good.